src/Timing.cc

```diff
@@ -385,4 +386,32 @@ float Timing::dynamicPower(odb::dbInst* inst, sta::Corner* corner)
   return (power.internal() + power.switching());
 }
 
+void Timing::makeEquivCells()
+{
+  auto app = OpenRoad::openRoad();
+  rsz::Resizer* resizer = app->getResizer();
+  resizer->makeEquivCells();
+}
+
+std::vector<odb::dbMaster*> Timing::equivCells(odb::dbMaster* master)
+{
+  sta::dbSta* sta = getSta();
+  sta::dbNetwork* network = sta->getDbNetwork();
+  sta::Cell* cell = network->dbToSta(master);
+  std::vector<odb::dbMaster*> masterSeq;
+  if (cell) {
+    sta::LibertyCell* libcell = network->libertyCell(cell);
+    sta::LibertyCellSeq* equiv_cells = sta->equivCells(libcell);
+    if (equiv_cells) {
+      for (sta::LibertyCell* equiv_cell : *equiv_cells) {
+        odb::dbMaster* master_
+            = reinterpret_cast<odb::dbMaster*>(equiv_cell->extCell());
```

Comment:
Use dbNetwork::staToDb for conversion from LibertyCell to dbMaster.

Reply:
Fixed that!